android/src/main/java/com/RNProximity/RNProximityPackage.java

```diff
@@ -16,7 +16,7 @@
       return Arrays.<NativeModule>asList(new RNProximityModule(reactContext));
     }
 
-    @Override
+    // Deprecated in RN 0.47.0
```

Comment:
Can you remove this comment?

Reply:
Why don't you just leave it here? For supporting under 0.47, the method `createJSModules` is needed. This comment can explain the reason why `createJSModules` is needed even though it is deprecated. Other popular libraries, such as code push, use this way :) (https://github.com/Microsoft/react-native-code-push/blob/5e332bbe838e02f4f25f2acf1056d1d7faedf1ba/android/app/src/main/java/com/microsoft/codepush/react/CodePush.java)

Reply:
I would rather avoid unnecessary comment and bump the version to `3.0.0` so people using an older version of RN can stay on `2.0.2`

Reply:
@williambout Okay, I have reflected your review!